dist/google-music.js

```diff
@@ -338,9 +352,9 @@ proto.hooks = {
             var durationStr = that.doc.getElementById(SELECTORS.playback.sliderId).getAttribute('aria-valuemax');
             var duration = parseInt(durationStr, 10);
 
-            title = (title) ? title.innerText : 'Unknown';
-            artist = (artist) ? artist.innerText : 'Unknown';
-            album = (album) ? album.innerText : 'Unknown';
+            title = (title) ? (title.innerText || title.textContent) : 'Unknown';
```

Comment:
Heh, didn't even know we were using `innerText`. We can prob drop `innerText` in favor of `textContent`. `innerText` is IE specific (IE <= 8) whereas `textContent` is more modern:

https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#Browser_compatibility